kasa/smartcamera.py

```diff
@@ -0,0 +1,130 @@
+"""Module for SmartCameras (EC70)."""
+from typing import Any, Dict, Optional
+
+from .credentials import Credentials
+from .modules import PTZ, Cloud, Time
+from .modules.ptz import Direction, Position
+from .smartcameraprotocol import SmartCameraProtocol
+from .smartdevice import DeviceType, SmartDevice, requires_update
+
+
+class SmartCamera(SmartDevice):
+    """Representation of a TP-Link Kasa Camera.
+
+    To initialize, you have to await :func:`update()` at least once.
+    This will allow accessing the properties using the exposed properties.
+
+    All changes to the device are done using awaitable methods,
+    which will not change the cached values, but you must await :func:`update()`
+    separately.
+
+    Errors reported by the device are raised as
+    :class:`SmartDeviceExceptions <kasa.exceptions.SmartDeviceException>`, and should be
+    handled by the user of the library.
+
+    Examples:
+        >>> import asyncio
+        >>> camera = KasaCam("127.0.0.1")
```

Comment:
Old name used here :-) Please add `test_camera_examples` to `kasa/tests/test_readme_examples.py` which will catch similar issues in the future.

Reply:
Can you clarify or suggest the fix here?

Reply:
The issue is that the name of the class is `SmartCamera` but the example uses `KasaCam`.

Reply:
Hopefullly I'm not to ignorant here, but is it using "KasaCam" because of the TYPE_TO_CLASS in the kasa/cli.py ?
https://github.com/python-kasa/python-kasa/blob/e2d8f52e7e13c979ed8c6705f2dc75beed985ba2/kasa/cli.py#L51

Reply:
```suggestion
        >>> camera = SmartCamera("127.0.0.1")
```
Is that what you are suggesting?